mlir/include/mlir/Dialect/MIOpen/utility/loweringUtils.h

```diff
@@ -47,6 +47,11 @@ inline int64_t calculateKBlockNum(int64_t n, int64_t ho, int64_t wo) {
   return gemmKBlocks;
 }
 
+/// Unwrap a value from the transforms surrounding it, gathering up the
+/// transforms
```

Comment:
This method deserves an example to help illustrate its semantic.
Nit: Ending with a period `.`

Reply:
And would it be better to make it returns a `std::tuple<Value, ArrayAttr>`? From the comment it is not clear `transforms` is also an output unless reading the implementation and the call sites.

Reply:
Taking a reference like that feels like an indication of being an out parameter, but I'm willing to go tuple

Reply:
Update: went with a tuple, it does make sense.